services/configuration/src/schema.rs

```diff
@@ -69,21 +108,41 @@ impl MaybeVote {
     }
 
     /// Creates a `Some` variant.
-    pub fn some(vote: Vote) -> Self {
+    pub fn some(vote: VotingDecision) -> Self {
         MaybeVote(Some(vote))
     }
+
+    /// Returns true if it's a `Some` variant hold `Vote`.
+    pub fn is_consent(&self) -> bool {
+        match self.0 {
+            Some(VotingDecision::Yea(_)) => true,
+            _ => false,
+        }
+    }
+}
+
+impl From<MaybeVote> for Option<VotingDecision> {
+    fn from(vote: MaybeVote) -> Option<VotingDecision> {
+        vote.0
+    }
+}
+
+impl From<Vote> for MaybeVote {
+    fn from(vote: Vote) -> MaybeVote {
+        MaybeVote::some(VotingDecision::Yea(vote))
+    }
 }
 
-impl From<MaybeVote> for Option<Vote> {
-    fn from(value: MaybeVote) -> Option<Vote> {
-        value.0
+impl From<VoteAgainst> for MaybeVote {
+    fn from(vote_against: VoteAgainst) -> MaybeVote {
+        MaybeVote::some(VotingDecision::Nay(vote_against))
     }
 }
 
 impl ::std::ops::Deref for MaybeVote {
```

Comment:
nitpick: let's `use std::ops::Deref` and avoid `::` at the beginning :upside_down_face: 
It might be considered unnecessary requirement, but our code style policy is pretty strict

Reply:
Done